<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveprocessingClass</class>
 <widget class="QMainWindow" name="CurveprocessingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curveprocessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_openfile">
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_edit">
        <property name="text">
         <string>编辑设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_operate">
        <property name="text">
         <string>操作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clear">
        <property name="text">
         <string>清除曲线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_setstyle">
        <property name="text">
         <string>曲线设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_lithologymap">
        <property name="text">
         <string>岩性图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_keeppara">
        <property name="text">
         <string>保留参数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_keepdata">
        <property name="text">
         <string>保留数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="myqchartview" name="my_chart" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X全局坐标：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="x_global">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y全局坐标：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y_global">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X窗口坐标：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="x_widget">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Y窗口坐标：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y_widget">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myqchartview</class>
   <extends>QWidget</extends>
   <header location="global">myqchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Curveprocessing.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>showDatawidget()</slot>
 </slots>
</ui>
